doc/source/10min.rst

```diff
@@ -719,7 +719,7 @@ of the columns with labels:
    df = df.cumsum()
 
    @savefig frame_plot_basic.png
-   plt.figure(); df.plot(); plt.legend(loc='best')
+   plt.figure(); df.plot(); plt.legend(loc='best')                 # noqa: E702
```

Comment:
@datapythonista If simply moved into single lines the graph remains empty.
```python
   @savefig frame_plot_basic.png
   plt.figure()
   df.plot()
   plt.legend(loc='best')
```
![ex2-afteri](https://user-images.githubusercontent.com/41163037/48912208-4f128c00-ee75-11e8-82bb-58826ad666eb.png)

This can be fixed by moving `@savfig` above the last statement:
```python
   plt.figure()
   df.plot()
   @savefig frame_plot_basic.png
   plt.legend(loc='best')
``` 
![ex2-afterii](https://user-images.githubusercontent.com/41163037/48912337-9862db80-ee75-11e8-96dc-8989601799e6.png)

However it becomes really verbose if you compare it to how it is before:
![ex2-before](https://user-images.githubusercontent.com/41163037/48912380-b9c3c780-ee75-11e8-8d5d-5ad7f303aa20.png)

Reply:
my preferred option would be to use:
```
plt.figure();
df.plot();
plt.legend(loc='best');
```
and I'm not sure if the `plt.figure()` is needed

Reply:
![single_line](https://user-images.githubusercontent.com/41163037/48945515-fc93a700-ef2a-11e8-9407-357fa686802e.png)